packages/grid/src/GridModel.ts

```diff
@@ -183,6 +185,38 @@ abstract class GridModel<
   ): IColumnHeaderGroup | undefined {
     return undefined;
   }
+
+  /**
+   * Find the links in the truncatedText including any links that may be truncated
+   * @param text The text in the cell
+   * @param truncatedText The truncated text in the cell (what is visible)
+   * @returns An array with link information
+   */
+  findLinksInVisibleText(
+    text: string,
+    truncatedText: string
+  ): ReturnType<typeof find> {
+    return this.getCachedLinksInVisibleText(text, truncatedText);
+  }
```

Comment:
Should the function return tokens for the entire cell then (including cut off text)? Since the truncated string is calculated in `GridRenderer`, I wasn't sure how to only return tokens for visible text without passing in the string.

Reply:
Yea, good point. For safety add another parameter for the `maxLength`, eg. `tokensForCell(column: ModelIndex, row: ModelIndex, maxLength?: number): Token[]`.